standard/docs/en/schema/merging.md

```diff
@@ -51,106 +52,163 @@ If the record is complete with versioning information, then the versioning secti
-* Retain any keys from **old** that are not mentioned in **new**
+If `omitWhenMerged` is set to `false`, ignore it.
 
-If the value of a key-value pair is a list of objects, the list merge rules below should be used; otherwise, a list should be treated as a single value.
+```eval_rst
+.. note::
 
-### Lists
+  The compiled release presently uses the same schema as the release schema, which means that the ``id``, ``date`` and ``tag`` fields are required in a compiled release. We invite discussion on whether to change these requirements in a separate compiled release schema in issue `#330 <https://github.com/open-contracting/standard/issues/330>`__, and on how to identify and date compiled and versioned releases in issue `#834 <https://github.com/open-contracting/standard/issues/834>`__.
 
-There are two merge patterns for lists of objects: identifier merge, and whole list merge. 
+  In the meantime, an intermediate solution is to set ``tag`` to ``["compiled"]``, ``date`` to the date of the most recent release, and ``id`` to ``{ocid}-{date}``, like in the `reference implementation <#reference-implementation>`__ of the merge routine.
+```
 
-#### Identifier merge
+### Versioned values
 
-When a list contains objects with their own `id` field, then:
+To convert a field's value in a release to a **versioned value**:
 
-* Check for an object in **old** with the same `id` as an object in **new**, and, if so, follow the object merge rules as above
-* If there is no object in **new** with the same `id` as an object in **old**, keep the object from **old** in the list
-* If there is an object in **new** with an `id` not found in **old** then add the object to the list
+1. Create an empty JSON object
+1. Set its `releaseID`, `releaseDate`, `releaseTag` fields to the release's `id`, `date`, `tag` values
+1. Set its `value` field to the field's value in the release
 
-Note: To remove information from an old list entry its values must be explicitly set to `null`. 
+A **versioned value** thus describes a field's value in a specific release.
 
-#### Whole list merge
+For example, in the above worked example, the estimated value of the procurement was $1,000 in a release (`tender/value/amount` was `1000`). Following the steps above, the versioned value is:
 
-A list should be treated as a single value if: the list has `"wholeListMerge": true` in the schema; the objects in the list have no `id` field according to the schema; or the list can contain non-objects according to the schema.
+```json
+{
+  "releaseID": "ocds-213czf-000-00002-01-tender",
+  "releaseDate": "2016-01-01T09:30:00Z",
+  "releaseTag": [
+    "tender"
+  ],
+  "value": 1000
+}
+```
 
-i.e. if the list exists in **new**, the entire list in **new** will overwrite the list in **old**. 
+In a **versioned release**, with a few exceptions, a field's value is replaced with an array of versioned values, which should be in chronological order by `releaseDate`.
```

Comment:
This information is critical to provide to users and is well explained here :-) I suggest it be signposted earlier in the document in the introduction. I'll make a comment where I think it would be good to include this.